packages/node_modules/proxy-state-tree/src/index.ts

```diff
@@ -13,18 +13,16 @@ export type Mutation = {
 }
 
 class ProxyStateTree {
-  state: object
-  options: Options
-  pathDependencies: object
-  globalDependencies: Set<Function>
-  objectChanges: Set<string>
-  mutations: Mutation[]
-  currentMutations: Mutation[]
-  paths: Set<string>[]
-  status: STATUS
-  proxy: any
-  currentFlushId: number
-  constructor(state: object, options: Options = {}) {
+  pathDependencies: object = {}
```

Comment:
should be private as well, but used in tests.
good practice for tests not to test using internal implementation